core/src/main/scala/akka/kafka/scaladsl/PartitionAssignmentHandler.scala

```diff
@@ -41,6 +41,15 @@ trait PartitionAssignmentHandler {
    */
   def onAssign(assignedTps: Set[TopicPartition], consumer: RestrictedConsumer): Unit
 
+  /**
+   * Called when partitions have been reassigned to a different consumer.
```

Comment:
```suggestion
   * Called when partition metadata has changed and partitions no longer exist.  This can occur if a topic is deleted or if the leader's metadata is stale.
```

https://cwiki.apache.org/confluence/display/KAFKA/KIP-429%3A+Kafka+Consumer+Incremental+Rebalance+Protocol